cmd/soroban-rpc/internal/preflight/preflight.go

```diff
@@ -96,11 +96,30 @@ type PreflightParameters struct {
 }
 
 type Preflight struct {
-	Auth            []string // ContractAuths XDR in base64
-	Footprint       string   // LedgerFootprint XDR in base64
-	Result          string   // SCVal XDR in base64
-	CPUInstructions uint64
-	MemoryBytes     uint64
+	Auth             []string // ContractAuths XDR in base64
+	DiagnosticEvents []string // DiagnosticEvents XDR in base64
```

Comment:
Maybe we should just call this `Events`?

```suggestion
	Events []string // DiagnosticEvents XDR in base64
```

Reply:
Done